scripts/build.py

```diff
@@ -295,29 +295,52 @@ def _compare_file_count(first_dir_path, second_dir_path):
 
 
 def _compare_file_count_for_multiple_dirs(
-        first_dir_list, second_dir_path):
-    """Ensure that two dir's file counts match.
+        first_dir_list, second_dir_list):
+    """Ensure that the total count of files in all directories in the first
+    list matches the count of files in all the directories in the second list.
 
     Args:
        first_dir_list: list(str). List of directories to compare.
-       second_dir_path: str. Second directory to compare.
+       second_dir_list: list(str). List of directories to compare.
 
     Raises:
         ValueError: The source directory list does not have the same file
-            count as the target directory.
+            count as the target directory list.
     """
 
     first_dir_file_count = 0
```

Comment:
first_dir_list_file_count or first_dirs_file_count?

Reply:
Better still, use file_counts = [0, 0] and put a for loop around the dirwalk code, so that you don't repeat code.

Reply:
Done.